### PR TITLE
Add el8 modules repo to repoclosures

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: repoclosure/yum.conf [hunk 1]
```diff
@@ -11,6 +11,12 @@ assumeyes=1
 syslog_ident=repoclosure
 syslog_device=
 
+[el8-modules]
```
Comment (ekohl) [inline repoclosure/yum.conf]: Are these modules we built or modules from CentOS? If the former, then I'd expect to see that in the name.

Comment (zjhuntin) [reply repoclosure/yum.conf]: They are modules from CentOS that we flattened @pcreech is this the best way to explain?

Comment (pcreech) [reply repoclosure/yum.conf]: To be _super_ technical, it's:
BaseOS
AppStream
PowerTools

all flattened into a single repo, with modules flattened into a standard RPM repo.  Ruby 2.6 module has been removed

**Comment by zjhuntin** [comment]
[test redmine]

**Pull request merged**